code/nodes/saveGraph/utils.q

```diff
@@ -0,0 +1,138 @@
+\d .automl
+
+// Utility functions for saveGraph
+
+// Python functionality
+plt:.p.import`matplotlib.pyplot;
+
+// @kind function
+// @category saveGraphUtility
+// @fileoverview Create regression target distribution plot and save down locally
+// @param target    {float[]} All target values
+// @param savePath  {int} Path to where plots are to be saved
+// @return {null} Target distribution plot saved to appropriate location
+saveGraph.i.regTargetPlot:{[target;savePath]
```

Comment:
The regTargetPlot and classTargetPlot functions can probably be the same function with an if/else statement for `plt.hist` vs `plt.bar`. The functions appear to have a lot of repeated code and can probably both be titled "Target distribution". Alternatively take the last 5/6 lines out of the function as a utility to avoid the code repetition